posts/change-data-capture-cloudsql-postgres-with-apache-airflow-1.md

```diff
@@ -0,0 +1,1002 @@
+                "{{execution_date.month}}/"
+                "{{execution_date.day}}/"
+                f"{table}_"
+                "{{ts_nodash}}.csv"
+            )
+
+            export_table = CloudSqlCsvExportOperator(
+                task_id = f'export_{table}',
+                gcp_conn_id = gcp_conn_id,
+                database_instance = db_instance_id,
+                project_id = gcp_project_id,
+                database_name = db_name,
+                gcs_bucket = gcs_bucket,
+                gcs_key = gcs_raw_keypath,
+                offload = offload_export,
+                export_sql = f'cloud_sql_cdc/export_csv/{table}.sql',
+                pool = 'cloudsql_operations',
+                trigger_rule='one_success'
+            )
+            get_schema >> export_table >> complete_export  
+            
+        kickoff_dag >> start_export >> get_schema
+        complete_export >> complete_dag
+    ```
+    Now, we have a DAG that extracts change data from our tables at an hourly interval as soon as that interval completes. We can now deploy this to production.
+
+## Step 14: Deploy to Production
+
+Once you commit your changes to git and merge to branches as needed, you are ready to deploy this to production.
+
+### **Create an Airflow Deployment**
+
+To create an Airflow Deployment on Astronomer, log into [Astronomer Cloud](https://app.gcp0001.us-east4.astronomer.io/), open your Workspace, and click **New Deployment**.
+
+![https://assets2.astronomer.io/main/docs/deploying-code/v0.23-deployments.png](https://assets2.astronomer.io/main/docs/deploying-code/v0.23-deployments.png)
+
+## **Configure Your Airflow Deployment**
+
+Use the **New Deployment** menu to configure the following:
+
+- **Name**
+- **Description** (Optional)
+- **Airflow Version**: We recommend using the latest version.
+- **Executor**: We recommend starting with Local.
+
+![https://assets2.astronomer.io/main/docs/deploying-code/v0.23-new_deployment-config.png](https://assets2.astronomer.io/main/docs/deploying-code/v0.23-new_deployment-config.png)
+
+When you've finished, click **Create Deployment**.
+
+Once you've initialized your Deployment, give it a few moments to spin up. Afterwards, you'll have access to your Deployment dashboard:
+
+![https://assets2.astronomer.io/main/docs/deploying-code/v0.23-new_deployment-dashboard.png](https://assets2.astronomer.io/main/docs/deploying-code/v0.23-new_deployment-dashboard.png)
+
+From this dashboard, you can:
+
+- Modify your Resources
+- Initialize the Airflow upgrade process
+- Configure **Email Alerts**
+- Access the Airflow UI
+- Access your Celery Dashboard (if using CeleryExecutor)
+- Delete your Deployment
+
+For more information on deployment configuration, read [Configure an Airflow Deployment on Astronomer](https://www.astronomer.io/docs/cloud/stable/deploy/configure-deployment/).
+
+## **Deploy Code from the CLI**
+
+### **a. Authenticate via the Astronomer CLI**
```

Comment:
Same Here

Reply:
Also, does it make sense to number these steps instead of letter them? That may be more intuitive.